.github/workflows/subgraph.yml

```diff
@@ -0,0 +1,35 @@
+name: Subgraph
```

Comment:
I changed the base branch to the branch from #117 but I'm still seeing the diff from that PR.  Technically this file and the next file should be the only changes in this PR right?

Reply:
We can change it back to `develop` once we merge the other PR I guess. But should make sure the diff is right because I can't think of why it would show these commits/diff here.

Reply:
Changed the base back to `develop` but still see the diffs. Since 117 was merged into this, this branch carries those changes, too.
I think we can go ahead and merge since the only other changes except for the ones that were from 117 are the GH action file changes.
But I can cherry-pick if that's what you prefer.

Reply:
Yeah let's cherry pick into new branch based off develop once 117 is merged so that it's clean.  I know it's an extra step but we know if we do it clean the diff shouldn't show and this will help keep things very clean :pray: